solidity/test/rewards/TestBeaconRewards.js

```diff
@@ -238,6 +238,38 @@ describe('BeaconRewards', () => {
             expect(unallocatedInKeep).to.eq.BN(19404000)
         })
 
+        it("should not count terminated groups in batch when distributing rewards", async () => {
```

Comment:
I think `should not count groups terminated in a batch when...` would be clearer.

Reply:
how about: `should not count a batch of terminated groups when distributing rewards`? `batch` is often followed by `of`

Reply:
Works for me.